pbspro.spec

```diff
@@ -50,9 +50,11 @@ Name: %{pbs_name}
 Version: %{pbs_version}
 Release: %{pbs_release}
 Source0: %{pbs_dist}
+%if 0%{?opensuse_bs}
```

Comment:
Is this change related to the ticket being addressed in this pull request? If not, I suggest we create another ticket to address this.